student.rb

```diff
@@ -1,6 +1,9 @@
 require_relative 'person'
+require_relative 'classroom'
```

Comment:
- [ optional] you don't need to require classroom here for more details check the above comment